README.adoc

```diff
@@ -45,3 +45,16 @@ Deployment Guide::
 
 .*No commits to master*
 NOTE: Please do not make any commits to the `master` branch. `master` is reserved for releases only. 
+
+Building documentation
+----------------------
+
+If you're contributing to the cloud documentation in this repo and want to build using our DAPs tooling, see the DAPs Quickstart for more information: http://docserv.nue.suse.com/documents/DAPS/daps-quick/html/
```

Comment:
* docserv is internal but we have public docs at: https://opensuse.github.io/daps/
* the spelling is "DAPS" with a capital for "Suite"